### PR TITLE
added GlobalExceptionHandler, custom validation annotations

### PR DESCRIPTION
**Pull request by citruscaviar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,6 +81,11 @@
 			<artifactId>liquibase-core</artifactId>
 			<version>${liquibase.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator</artifactId>
+			<version>${hibernate-validator.version}</version>
```
Comment (Elena-Bruyako) [inline pom.xml:87]:
```suggestion
			<groupId>org.springframework.boot</groupId>
                         <artifactId>spring-boot-starter-validation</artifactId>
```

File: src/main/java/com/onlinebookstore/store/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+    @NotBlank
     private String author;
+    @NotBlank
+    @Isbn
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotBlank
+    @Description
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/store/dto/CreateBookRequestDto.java:24]:
```suggestion
    @Size
```

don't need to create custom annotation to validate size

File: src/main/java/com/onlinebookstore/store/validation/DescriptionValidator.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.onlinebookstore.store.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class DescriptionValidator implements ConstraintValidator<Description, String> {
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/store/validation/DescriptionValidator.java:6]: remove

File: src/main/java/com/onlinebookstore/store/validation/Description.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = DescriptionValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Description {
```
Comment (Elena-Bruyako) [inline src/main/java/com/onlinebookstore/store/validation/Description.java:13]: remove

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

**Review by liliia-ponomarenko — APPROVED** [review]
Awesome 😎

**Pull request merged**